tasks/run_agent.sh

```diff
@@ -4,12 +4,12 @@
 declare PT__installdir
 source "$PT__installdir/deploy_pe/files/common.sh"
 PUPPET_BIN=/opt/puppetlabs/bin
+retries=${PT_retries:-5}
```

Comment:
I wonder why shellcheck doesn't flag this as needing to be quoted?  I guess it's ok in this context.

Reply:
I think it is because if it does not exist, we are setting the default.